simplq/src/components/common/Button/Button.stories.jsx

```diff
@@ -10,24 +10,16 @@ export default {
 };
 
 export const CreateQButton = (props) => {
-  return (
-    <StandardButton onClick={props.onClick} outlined>
-      Create Queue
-    </StandardButton>
-  );
+  return <StandardButton onClick={props.onClick}>Create Queue</StandardButton>;
 };
 
```

Comment:
We used the wrong className. Outlined means it shouldn't be colored, but we had it the other way around.

Reply:
ha yes, my bad.